### ISSUE TITLE
Dead links to CrashAggregates 

### ISSUE THREAD

**Issue by harterrt** [mozilla/firefox-data-docs#267]
The following links are being flagged by the Dead Link CI. 

```
FILE: src/datasets/other/hgpush/reference.md
[✖] https://github.com/mozilla/telemetry-batch-view/blob/master/docs/CrashAggregateView.md → Status: 404
...
FILE: src/datasets/pings.md
[✖] https://github.com/mozilla/telemetry-airflow/blob/master/dags/crash_aggregates.py → Status: 404

**Comment by harterrt** [comment]
Looks like this was caused by https://github.com/mozilla/telemetry-batch-view/pull/530

**Comment by mreid-moz** [comment]
Fixed by #268
